Tests/Unit/SOGoTest.h

```diff
@@ -54,6 +54,8 @@
 
 - (BOOL) run;
 
+NSString *_stringForCharacterAtIndex(NSUInteger index, NSString *str, NSUInteger length);
```

Comment:
We shouldn't need this one since it's only used internally to the SOGoTest.m file.

Are you getting a warning otherwise?